webpack.config.js

```diff
@@ -40,10 +38,20 @@ const webpackConfig = {
     path: path.join(__dirname, "assets/build"),
     filename: "[name].js",
     publicPath: "/assets/build"
+  },
+
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        loader: "babel-loader",
+        include: /node_modules\/react-aria-components/
+      }
```

Comment:
Unfortunately, we still need babel to process react-aria-components, as it's non-transpiled ES6/React.

Reply:
we might have a good way to do this here: https://github.com/devtools-html/devtools-core/blob/master/packages/devtools-launchpad/webpack.config.js#L33-L41

Reply:
Ah, I did see that but wasn't sure we wanted to add a special case there. If we're cool with that, I can certainly open up a PR for that.

Reply:
Updated devtools-launchpad in https://github.com/devtools-html/devtools-core/pull/1075. I'll update this PR to pull in that branch for testing.